### DISCUSSION TITLE
keyball44のレイヤー3の特殊キーコードについて

### DISCUSSION THREAD

**Discussion by 223mle** [Yowkees/keyball#401] [Q&A] [answered]
[こちら](https://github.com/Yowkees/keyball/blob/main/qmk_firmware/keyboards/keyball/lib/keyball/keycodes.md#japanese)に記載してあるような、レイヤー３でカーソル速度やスクロール速度の設定する特殊キーボードが自分のRemap上にありません。  
かわりにAnyというキーがあります。どのようにカーソル速度やスクロール速度を設定すれば良いのでしょうか？

**Comment by Yowkees** [comment]
ご連絡ありがとうとうございます。

Remap下側のキーがズラッと並んでいる中の、「FUNCTIONS」タブの「VIA USER KEY」の欄のKb1,Kb1,Kb2,・・・が特殊キーになりますがありませんか？
もしくは、変更したいキーをクリックして、Keycode欄に"kb"と入力していただくと、下にリストが表示されるかと思いますのでお試しください。

以上、よろしくお願いいたします。

**Comment by 223mle** [comment]
ご回答ありがとうございます。
以下の画像のように、Kbの特殊キーはなく、kbと入力してもリストが表示されない状態となっております。  
Remap for QMK Firmware 0.18.17 or lower

![image](https://github.com/Yowkees/keyball/assets/69502527/2e1b191c-bb8a-4e10-93bd-6448099bab79)

**Comment by Yowkees** [comment, marked as answer]
旧バージョンのリマップを使用しているのですね。
REMAPのキーボードカタログにアップしている最新のKeyballファームウェアを書き込み直していただくと、最新のREMAPが使用可能です。新しいREMAPではkbの特殊キーが使用できます。

以上、お手数ですがご対応お願いいたします。

**Reply by 223mle** [reply]
できました！
丁寧かつ適切な対応、ありがとうございました🙇
